source/change-streams.rst

```diff
@@ -426,7 +426,17 @@ startAtClusterTime
 
 :since: 4.0
 
-``startAtClusterTime`` specifies that a ``changeStream`` will only return changes that occurred after the specified ``$clusterTime``.
+``startAtClusterTime`` specifies that a ``changeStream`` will only return changes that occurred after the specified ``Timestamp``.
+
+The server expects ``startAtClusterTime`` as a bson document of the form
+
+.. code:: typescript
+
+    {
+      ts: Timestamp;
+    }
+
+Drivers MUST allow users to specify a ``startAtClusterTime`` option in the ``Collection.watch`` and ``Database.watch`` helpers. They SHOULD allow users to specify this value as a raw ``Timestamp``, and SHOULD NOT force users to wrap the timestamp as the server specifies.
```

Comment:
I suppose this jives with discussion in [SERVER-34818](https://jira.mongodb.org/browse/SERVER-34818). I did ask if the server team would reconsider the "ts" wrapping (as they admitted it's now superfluous), but this is correct given the current implementation. :+1: